LearnAndCodeAssignment_Week1/GettingNeighbouringCountries.java

```diff
@@ -0,0 +1,51 @@
+package LearnAndCodeAssignment_Week1;
+
+import java.util.Scanner;
+
+public class GettingNeighbouringCountries {
+	public static String getNeighboringCountries(String countryCode) {
+        switch (countryCode) {
+            case "IN":
```

Comment:
hardcoded strings/values shouldn't be used in code either create meaningful variables or define constants. Please read more about it.